common/models/application.js

```diff
@@ -108,7 +108,9 @@ module.exports = function(Application) {
    * @param {String} owner Owner's user ID.
    * @param {String} name  Name of the application
    * @param {Object} options  Other options
-   * @param {Function} callback  Callback function
+   * @callback {Function} callback  Callback function
+   * @param {Error} err
```

Comment:
@hacksparrow  Is it necessary to add `@param {Error} err` to callbacks that have at most one argument? I.e. `cb(err)` on error, `cb()` on success.

I would prefer if strong-docs was able to add this automatically.